board/safety.h

```diff
@@ -91,6 +97,10 @@ int safety_fwd_hook(int bus_num, CAN_FIFOMailBox_TypeDef *to_fwd) {
   return current_hooks->fwd(bus_num, to_fwd);
 }
 
+int safety_relay_hook(int to_set) {
+  return current_hooks->relay(to_set);
```

Comment:
no reason to pass along the input `to_set`, right? this function and the relay hooks can all have `void` as input

Reply:
Sure. This was done to match the style of the other safety hooks.

Reply:
you mean we have safety hooks with unused input?

Reply:
Just remove it